src/vcSettingsUI.cpp

```diff
@@ -368,6 +369,22 @@ void vcSettingsUI_Show(vcState *pProgramState)
           ImGui::Combo(vcString::Get("scenePOILabelSize"), &pProgramState->settings.tools.label.textSize, labelSizeOptions, (int)udLengthOf(labelSizeOptions));
         }
 
+        if (pProgramState->activeSetting == vcSR_UnitsOfMeasurement)
+        {
+          static char unitTextBuffer[128] = {};
+          if (ImGui::Button(vcString::Get("settingsSetMeasurementSystemMetric")))
+          {
+            vcUnitConversion_SetMetric(&pProgramState->settings.unitConversionData);
+            udStrcpy(unitTextBuffer, vcString::Get("settingsUnitsOfMeasurementUsingMetric")); //Temp feedback for the user until we implement per unit options
+          }
+          if (ImGui::Button(vcString::Get("settingsSetMeasurementSystemImperial")))
+          {
+            vcUnitConversion_SetImperial(&pProgramState->settings.unitConversionData);
+            udStrcpy(unitTextBuffer, vcString::Get("settingsUnitsOfMeasurementUsingImperial")); //Temp feedback for the user until we implement per unit options
+          }
+          ImGui::Text("%s", unitTextBuffer);
+        }
```

Comment:
I know this is for future expansion but right now with just these 2 options, is should be a combo box on the "Appearance" tab